CHAPTER3-COMMIT CONVENTIONS.md

```diff
@@ -0,0 +1,143 @@
+# Chapter 3 - Commit Message Conventions
+
+These rules are adopted from [the AngularJS commit conventions](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#commits).
+
+* [Subject line](#subject-line)
+    * [Allowed `<type>`](#allowed-type)
+    * [Allowed `<scope>`](#allowed-scope)
+    * [`<subject>` text](#subject-text)
+* [Message body](#message-body)
+* [Message footer](#message-footer)
+    * [Breaking changes](#breaking-changes)
+    * [Referencing issues](#referencing-issues)
+* [Examples](#examples)
+
+
+The format of a commit message should be as follows:
+
+```
+<type>(<scope>): <subject>
+<BLANK LINE>
+<body>
+<BLANK LINE>
+<footer> (optional)
+```
+
+Any line of the commit message should not exceed 100 characters. This allows the message to be easier to read.
```

Comment:
"Any line of **a** commit message should not exceed 100 characters. This allows **for** the message to be easier to read."